Quasardb/GetBytes.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace Quasardb
+{
+    /// <summary>
+    /// Helper class
+    /// </summary>
+    public static class Helper
+    {
+        /// <summary>
+        /// Safely Get Bytes
+        /// </summary>
+        public static byte[] GetBytes(int size, IntPtr Pointer)
+        {
+            // CAUTION: limited to 32 bits!!!
+            if (size <= 0)
+                return null;
+            if (Pointer == null)
```

Comment:
Euuh, shouldn't you compare with `IntPtr.Zero`?

Reply:
good catch